### PR TITLE
a23-stella-studio

### PR DESCRIPTION
**Pull request by ghost**
Creation of stella studio

### CODE REVIEW

File: src/jm/html-css/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <div class="buttons">
+          <button class="uno">Get Started</button>
+          <button class="dos">Get Quote</button>
+        </div>
+        <div class="services">
+            <ul>
+                <li><a href="">/Web Design</a></li>
+                <li><a href="">/Branding</a></li>
+                <li><a href="">/Marketing</a></li>
+              </ul>
```
Comment (Iregui44) [inline src/jm/html-css/index.html:47]: There is something wrong wiht identation or closing tags here

File: src/jm/html-css/style.css [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+  border: none;
+  background-color: #ffffff;
+  color: #000000;
+  font-size: 20px;
+  border-radius: 50px;
+}
+
+@media (max-width: 768px) {
+  .circle {
+    padding: none;
```
Comment (Iregui44) [inline src/jm/html-css/style.css:146]: none is not a valid value for this attributes.

File: src/jm/html-css/index.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>A23 STELLA STUDIO</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
```
[the remark this answers is not in the document]
Comment (Juanmediina) [reply src/jm/html-css/index.html]: correction made

File: src/jm/html-css/style.css [hunk 4]
```diff
@@ -0,0 +1,203 @@
+body {
```
[the remark this answers is not in the document]
Comment (Juanmediina) [reply src/jm/html-css/style.css]: file name has been changed

File: src/jm/html-css/style.css [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+  font-size: 20px;
+}
+
+.founder-info .p2 {
+  margin: 0;
+  font-weight: 100;
+  font-size: 15px;
+}
+
+.imgs {
```
[the remark this answers is not in the document]
Comment (Juanmediina) [reply src/jm/html-css/style.css]: the image appears after the details

File: src/jm/html-css/index.html (hunk 1 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: corrections made

File: src/jm/html-css/style.css (hunk 2 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css]: changed to value=0

**Review by Iregui44 — APPROVED** [review]

**Comment by Iregui44** [comment]
You have conflicting errors please fix them to be able to marge

**Pull request closed without merging**